app/components/moderation-list-row/component.js

```diff
@@ -79,6 +83,14 @@ export default Component.extend({
         this.get('fetchData').perform();
     },
 
+    formattedDateLabel(rawDate) {
+        const gtDayAgo = moment().diff(rawDate, 'days') > 1;
+        const formattedDate = gtDayAgo ?
+            moment(rawDate).format('MMMM DD, YYYY') :
+            moment(Math.min(Date.parse(rawDate), Date.now())).fromNow();
```

Comment:
If we're using `moment`, it would be nice to use it consistently instead of `Date`. I think it would make for more easily readable code.